src/test/java/bio/terra/flight/ProfileUpdateFlightTest.java

```diff
@@ -2,35 +2,31 @@
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.contains;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.mock;
 
 import bio.terra.common.FlightTestUtils;
 import bio.terra.common.category.Unit;
+import bio.terra.model.BillingProfileUpdateModel;
 import bio.terra.service.job.JobMapKeys;
 import bio.terra.service.profile.flight.update.ProfileUpdateFlight;
 import bio.terra.stairway.FlightMap;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
 import org.mockito.Mock;
-import org.mockito.junit.MockitoJUnitRunner;
 import org.springframework.context.ApplicationContext;
 import org.springframework.test.context.ActiveProfiles;
 
-@RunWith(MockitoJUnitRunner.StrictStubs.class)
 @ActiveProfiles({"google", "unittest"})
-@Category(Unit.class)
-public class ProfileUpdateFlightTest {
+@Tag(Unit.TAG)
+class ProfileUpdateFlightTest {
 
-  @Mock private ApplicationContext context;
   @Mock private FlightMap map;
-  @Mock private bio.terra.model.BillingProfileUpdateModel profileMock;
 
   @Test
-  public void testConstructFlight() {
-    when(map.get(JobMapKeys.REQUEST.getKeyName(), bio.terra.model.BillingProfileUpdateModel.class))
-        .thenReturn(profileMock);
-    var flight = new ProfileUpdateFlight(map, context);
+  void testConstructFlight() {
+    FlightMap map = new FlightMap();
+    map.put(JobMapKeys.REQUEST.getKeyName(), new BillingProfileUpdateModel());
+    var flight = new ProfileUpdateFlight(map, mock(ApplicationContext.class));
```

Comment:
This is an editorial change that I could revert. My thinking is mocking comes in two flavors; in one case an API is mocked because it's required for the test to run; in the other case it's required and we either need to provide a stub, or verify its use. Using `mock()` inline is a way to tell that reader that this mock is never verified and and its methods are never stubbed.